pages/phases/getting-started/measuring-a-hiring-action.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: article
+permalink: /hiring-phases/getting-started/collecting-data/
+section: hiring-phases
+title: Collecting data
+sidenav: hiring-phases
+phase: getting-started
+sub-phase-order: 030
+---
+
+## Collecting baseline data
+
+<p class="usa-intro">
+  To gauge the success of your hiring action, collect baseline data about recent delegated examination (DE) hiring actions at your agency. You'll compare this information with the results you receive when your new hiring action is complete.
+</p>
+
+For one to two recent DE actions, gather:
```

Comment:
should we also specify that the baseline data should be for a similar GS level and job category (ie: 2210)?